packages/langium-cli/src/generator/module-generator.ts

```diff
@@ -25,14 +25,16 @@ export function generateModule(grammars: Grammar[], config: LangiumConfig, gramm
             import type { LanguageMetaData } from '../../languages/language-meta-data${config.importExtension}';
             import { ${config.projectName}AstReflection } from '../../languages/generated/ast${config.importExtension}';
             import type { Module } from '../../dependency-injection${config.importExtension}';
-            import type { LangiumGeneratedServices, LangiumGeneratedSharedServices, LangiumSharedServices, LangiumServices } from '../../services${config.importExtension}';
+            import type { LangiumGeneratedServices, LangiumGeneratedSharedServices } from '../../services${config.importExtension}';
+            import type { LangiumSharedServices, LangiumServices } from '../../lsp/lsp-services${config.importExtension}';
         `.appendTemplateIf(hasIParserConfigImport)`
 
             import type { IParserConfig } from '../../parser/parser-config${config.importExtension}';
         `
     ).appendTemplateIf(!config.langiumInternal)`
 
-        import type { LangiumGeneratedServices, LangiumGeneratedSharedServices, LangiumSharedServices, LangiumServices, LanguageMetaData, Module${hasIParserConfigImport ? ', IParserConfig' : ''} } from 'langium';
+        import type { LangiumGeneratedServices, LangiumGeneratedSharedServices, LanguageMetaData, Module${hasIParserConfigImport ? ', IParserConfig' : ''} } from 'langium';
+        import type { LangiumSharedServices, LangiumServices } from 'langium/lsp';
```

Comment:
I started with the core services, and then switched to the full ones, but I can't remember why. I think I had an issue with one of the generated examples, but it may have been pre-rebase. I'll go back over this and try the core service set again.